cluster/src/test/java/io/scalecube/cluster/ClusterTest.java

```diff
@@ -26,130 +26,144 @@ public void testJoinDynamicPort() throws Exception {
     // Start seed node
     Cluster seedNode = Cluster.joinAwait();
 
-    // Start other nodes
     int membersNum = 10;
-    long start = System.currentTimeMillis();
-    List<Cluster> otherNodes = new ArrayList<>();
-    for (int i = 0; i < membersNum; i++) {
-      otherNodes.add(Cluster.joinAwait(seedNode.address()));
+    List<Cluster> otherNodes = new ArrayList<>(membersNum);
```

Comment:
Somethings weird shown here... you should not have diff at that class, since this code is already in the master.